ckanext/switzerland/helpers.py

```diff
@@ -302,3 +302,35 @@ def get_translated_group_title(titles_string):
 # See: http://stackoverflow.com/a/518232
 def strip_accents(s):
    return ''.join(c for c in unicodedata.normalize('NFD', s) if unicodedata.category(c) != 'Mn')  # noqa
+
+
+def map_to_valid_format(resource_format):
+    format_mapping = {
+        'CSV': ['csv', 'aspx'],
+        'GeoJSON': ['geojson'],
+        'GeoTIFF': ['geotiff'],
+        'GPKG': ['gpkg'],
+        'HTML': ['html'],
+        'INTERLIS': ['interlis'],
+        'JSON': ['json'],
+        'KMZ': ['kmz'],
+        'MULTIFORMAT': ['multiformat'],
+        'ODS': ['ods', 'vnd.oas...', 'pc-axis file', 'Sparql-...', 'ESRI Shapefile'], # noqa
```

Comment:
Check the correct mapping in the Formate.xls, at least PC-AXIS and ESRI-Shapefile have their own format